dot/parachain/availability-distribution/availability_distribution_test.go

```diff
@@ -248,3 +251,176 @@ func TestHandlePoVFetchingRequest(t *testing.T) {
 		wg.Wait()
 	})
 }
+
+func Test_getBlockAncestorsInSameSession(t *testing.T) {
```

Comment:
Can you please rename this to either `TestGetBlockAncestorsInSameSession` or `TestAvailabilityDistribution_getBlockAncestorsInSameSession`